Cython/Compiler/MatchCaseNodes.py

```diff
@@ -51,15 +54,16 @@ def refactor_cases(self):
             if c is not None and c.is_simple_value_comparison():
                 body = SubstitutedIfStatListNode(
                     c.body.pos,
-                    stats = c.body.stats,
-                    match_node = self
+                    stats=c.body.stats,
+                    match_node=self
                 )
                 if_clause = Nodes.IfClauseNode(
                     c.pos,
                     condition=c.pattern.get_simple_comparison_node(subject),
                     body=body,
                 )
-                assignments = c.pattern.generate_target_assignments(subject)
+                # Passing None for env is safe only because we know it's a simple value comparison.
+                assignments = c.pattern.generate_target_assignments(subject, None)
```

Comment:
This `None` seems a bit dubious to me. It works and is designed to work, but I wonder if there's a better way.